openshift/07-apicast-cloud-hosted-build-config-policy-disabler.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: apicast-cloud-hosted
+  name: apicast-cloud-hosted-policy-disabler
+spec:
+  nodeSelector: null
+  postCommit:
+    args:
+      - '--test'
+      - '--lazy'
+    command:
+      - bin/apicast
+  output:
+    to:
+      kind: ImageStreamTag
+      name: apicast-cloud-hosted-policy-disabler:latest
```

Comment:
Or maybe base this on raw apicast:master and push to apicast:saas-master tag ? 
And then base the other build on apicast:saas-master ?

Reply:
I Like the previous idea better, everything would be within the APIcast cloud hosted project and, for me at least, this would minimize the complexity. Unless of course, you prefer this method.

Reply:
done in 75b91358f6c68284ce47f22d144fc86551addb7a